src/elements/TableLinkCell/TableLinkCell.js

```diff
@@ -37,7 +37,13 @@ const TableLinkCell = ({
             <i className={item.state} />
           </Tooltip>
         )}
-        <div className="name_status_row">{data.value}</div>
+        <div className="name_status_row">
+          {data.value}
+          {link.match(/functions/) && (
+            <span>{item.tag ? item.tag : 'latest'}</span>
```

Comment:
This is equivalent to this:
```suggestion
            <span>{item.tag || 'latest'}</span>
```